File: java/test/_pid_neuron.java
import liba.pid_neuron;

class _pid_neuron {
    public static void main(String[] args) {
        pid_neuron obj = new pid_neuron();
        obj.kpid(10, 1, 0.1, 1).wpid(1, 0, 0).zero();
        System.out.print(obj.inc(1, 0) + " ");
        System.out.println(obj.off(1, 0));
        obj.k(obj.k());
        obj.kp(obj.kp());
        obj.ki(obj.ki());
        obj.kd(obj.kd());
        obj.wp(obj.wp());
        obj.wi(obj.wi());
        obj.wd(obj.wd());
        obj.outmax(obj.outmax());
        obj.outmin(obj.outmin());
        obj.out();
        obj.fdb();
        obj.err();
        obj.ec();
    }
}

File: javascript/test/pid_neuron.js
#!/usr/bin/env node
var liba = require("../liba")
var ctx = new liba.pid_neuron()
ctx.wpid(1, 0, 0).kpid(10, 1, 0.1, 1)
console.log(ctx.zero().off(10, 0))
console.log(ctx.zero().inc(10, 0))
ctx.k = ctx.k
ctx.kp = ctx.kp
ctx.ki = ctx.ki
ctx.kd = ctx.kd
ctx.wp = ctx.wp
ctx.wi = ctx.wi
ctx.wd = ctx.wd
ctx.summax = ctx.summax
ctx.outmax = ctx.outmax
ctx.outmin = ctx.outmin
ctx.out
ctx.fdb
ctx.err
ctx.ec
ctx.delete()
